### ISSUE TITLE
Allow Enterprise Managers to add Shipping Notes in Edit Orders screen

### ISSUE THREAD

**Issue by lin-d-hop** [openfoodfoundation/openfoodnetwork#4146]
## Description

When Users are checking out they can leave a Shipping Note for the Enterprise  Manager:
![Screenshot from 2019-08-12 16-18-31](https://user-images.githubusercontent.com/8669848/62876448-e332c500-bd1c-11e9-8883-4a69ade35e10.png)

Back in v1 it was possible for the Enterprise Manager to update an Order with a Shipping Note. However in v2 this functionality has been lost and there is no workaround. 

## Expected Behavior

Hub Managers can add a Shipping Note to an order in the edit orders screen.

## Actual Behaviour

There is no longer an option to add a shipping note.

## Context

This has meant that a previously internal OFN feature is being done with scraps of paper for a significant UK food hub, adding mistakes to their process which can be costly.

## Your Environment

* Version used: v2.2.2
* OFN Platform instance: UK

## Possible Fix

With v2 a Tracking Number field has been added to the Orders screen:
![Screenshot from 2019-08-12 16-30-29](https://user-images.githubusercontent.com/8669848/62877274-9354fd80-bd1e-11e9-8021-ab4048458450.png)
![Screenshot from 2019-08-12 16-30-45](https://user-images.githubusercontent.com/8669848/62877280-96e88480-bd1e-11e9-97fc-e1ae7dc717d5.png)

An ideal solution would be to re-purpose this field to Shipping Notes and have the contents read and write to the same field that the customer can write to at Checkout.

**Comment by luisramos0** [comment]
I dont think we need to repurpose an existing spree field to do something else. I think we have a better option.

The customer currently can write a shipping note on checkout, this is stored in a field called  order.special_instructions.
We can display this field in the order edit page (as "Shipping Notes") and make it editable by the manager. This will be stored in order.special_instructions. ok?

We can display this below tracking or replace tracking.

**Comment by lin-d-hop** [comment]
Yep that sounds perfect. 
My intention on 're-purposing' the field was to keep the UI clean.
So replacing the 'tracking' field with this field is totally perfect :-)

**Comment by lbwright22** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Invoices & Pick Lists](https://trello.com/c/Vn1V1M9M/912-invoices-pick-lists)

**Comment by RachL** [comment]
Hub managers can now add notes to orders. @lin-d-hop @BethanOFN is it enough to close this one or do you still need orders to display customers notes on the edit order screen?
